### PR TITLE
Revert alert cron job

### PR DESCRIPTION
**Pull request by mikeshultz**
### Description:

Added an origin-cron job that will alert us on transaction reverts in the last 24 hours.

### Checklist:

- [x] Test your work and double-check to confirm that you didn't break anything
- [ ] [Wrap any new text/strings with `fbt`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#wrapping-text) for translation
- [ ] If there are any new/changed strings to translate, [`npm run translate`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#integrating-translations)
- [ ] Update any relevant READMEs and [docs](https://github.com/OriginProtocol/origin/tree/master/docs)

### CODE REVIEW

File: infra/cron/src/jobs/revertAlertJob.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+  sendgridMail.setApiKey(process.env.SENDGRID_API_KEY)
+
+  const apiKey = process.env.ETHERSCAN_API_KEY
+
+  let total = 0
+  const reverts = []
+  const contracts = []
+  contracts.push({
+    address: contractsJson['Marketplace'],
```
Comment (franckc) [inline infra/cron/src/jobs/revertAlertJob.js]: It's not Marketplace_V00 and MarketplaceEpoch_V00 ?

File: infra/cron/src/jobs/revertAlertJob.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    for (const tx of internals) {
+      total += 1
+      if (tx.isError === '1') {
+        reverts.push(tx.hash)
+      }
+    }
+  }
+
+  // test revert
+  // reverts.push('0xdeadbeef')
```
Comment (franckc) [inline infra/cron/src/jobs/revertAlertJob.js]: nit: remove

File: infra/cron/src/jobs/revertAlertJob.js (hunk 1 above)
Comment (mikeshultz) [reply infra/cron/src/jobs/revertAlertJob.js]: Nope

https://github.com/OriginProtocol/origin/blob/74650f8eb3c14cbb0934e7ecce6df849631429bf/packages/contracts/build/contracts_rinkeby.json#L2-L5

File: infra/cron/src/jobs/revertAlertJob.js (hunk 2 above)
Comment (mikeshultz) [reply infra/cron/src/jobs/revertAlertJob.js]: eh, useful for testing, kind of left it intentionally for later testing.

**Comment by mikeshultz** [comment]
> Would be nice to save the last checked block and check forward from that point but we can add that later if this doesn't work.

Agree.  Could probably use redis here, even if it seems like overkill.

**Pull request merged**